index-handlers/src/main/java/no/sikt/nva/nvi/index/utils/LanguageLabelUtil.java

```diff
@@ -0,0 +1,31 @@
+package no.sikt.nva.nvi.index.utils;
+
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import nva.commons.core.ioutils.IoUtils;
+
+public final class LanguageLabelUtil {
```

Comment:
Maybe use https://github.com/BIBSYSDEV/nva-language-java, since this has the functionality